test/analysis.py

```diff
@@ -177,41 +181,40 @@ def setUp(self) -> None:
 
     def test_01_basic(self):
         pytorch_output = run_analysis(
-            ["autogroq", "linear_pytorch.py", "--analyze-only"]
+            ["benchit", "linear_pytorch.py", "--analyze-only"]
         )
         assert np.array_equal(pytorch_output, (1, 0, 0))
 
     def test_02_basic_keras(self):
-        keras_output = run_analysis(["autogroq", "linear_keras.py", "--analyze-only"])
+        keras_output = run_analysis(["benchit", "linear_keras.py", "--analyze-only"])
         assert np.array_equal(keras_output, (1, 0, 0))
 
     def test_03_depth(self):
         # Depth is only tested for Pytorch, since Keras has no max_depth support
         output = run_analysis(
-            ["autogroq", "linear_pytorch.py", "-d1", "--analyze-only"]
+            ["benchit", "linear_pytorch.py", "--max-depth", "1", "--analyze-only"]
         )
         assert np.array_equal(output, (2, 0, 0))
 
     def test_04_build(self):
         output = run_analysis(
-            ["autogroq", "linear_pytorch.py", "-d1", "--targets", "60931adb"]
+            ["benchit", "linear_pytorch.py:60931adb", "--max-depth", "1"]
```

Comment:
(same comment as before)
Looks like you are missing `--build-only`  here. This was the default behavior of autogroq. Without --build-only GitHub will probably get stuck here since it doesn't have access to the remote CPU.